test/EndToEndTests/Tests/Client/AsynchronousTests/AsynchronousQueryTests.cs

```diff
@@ -294,21 +295,25 @@ public void ExecuteBatchWithSaveChangesOptionsReturnsCorrectResults()
             this.EnqueueTestComplete();
         }
 
-#if !PORTABLELIB && !NETCOREAPP1_0
         /// <summary>
         /// CancelRequest for Batch Requests
         /// </summary>
-        // github issuse: #896
-        // [Fact, Asynchronous]
+        [Fact, Asynchronous]
         public void CancelBatchRequestTest()
         {
             var context = this.CreateWrappedContext<DefaultContainer>().Context;
             ServicePoint servicePoint = null;
             context.SendingRequest2 += ((sender, args) =>
             {
-                if (args.RequestMessage is HttpClientRequestMessage)
+                servicePoint = ServicePointManager.FindServicePoint(args.RequestMessage.Url);
+                MethodInfo ReleaseConns = servicePoint.GetType().GetMethod
+                        ("ReleaseAllConnectionGroups",
+                        BindingFlags.Instance | BindingFlags.NonPublic);
+                ReleaseConns.Invoke(servicePoint, null);
+
+                if (args.RequestMessage is HttpWebRequestMessage)

```

Comment:
This test is not possible when using `HttpClientRequestMessage`?

Reply:
fixed.